### ISSUE TITLE
Almost no outputs for the demo image after 450 iterations

### ISSUE THREAD

**Issue by 1ssb** [crockwell/pixelsynth#3]
![input_fs_image_R_0](https://github.com/crockwell/pixelsynth/assets/40661648/b1b4b708-6a62-4cbc-8db8-7c846503fd18)
![input_image_](https://github.com/crockwell/pixelsynth/assets/40661648/40ff32f5-ff71-4527-89a3-d7c0dde9c368)
![output_image_R_0](https://github.com/crockwell/pixelsynth/assets/40661648/8e31d34d-72f6-4708-8688-7c8f02312754)

Same as topic name. PFA the images.

**Comment by crockwell** [comment]
Hi -- thanks for your issue and suggested pull request. Could you please elaborate on your suggestion / concern? I don't quite follow.

**Comment by 1ssb** [comment]
The third image is the final output, I have followed the steps as recorded in the repository, can you reproduce your results as published with the new libraries/ can you update/publish the exact environment.yml whereby you achieved your results?

**Comment by crockwell** [comment]
Sorry to hear you are struggling. I did share my yml [here](https://github.com/crockwell/pixelsynth/blob/main/docs/INSTALL.md#requirements-used-by-paper-code), but if your output is totally black I would guess the environment is not the problem. I can confirm I ran the code and it works for me!

Is it possible the pretrained model [here](https://github.com/crockwell/pixelsynth/blob/main/docs/DEMO.md#pretrained-model) isn't actually being used during the forward pass? Or, the output image isn't being properly saved?

Also, are you using "sh scripts/demo_image.sh"?

After these checks, I'd recommend putting a breakpoint before the network output. Is everything 0? Is it very small and needs to be rescaled? Is the autoregressive model running on the non-visible areas?
